scripts/docker-compose.yml

```diff
@@ -100,17 +100,50 @@ services:
        - kafka
        - db-host-inventory
        - puptoo
-       - zookeeper
+       - createbuckets
+
+  insights-inventory-web:
+    <<: *inventory
+    command: "make upgrade_db run_inv_web_service"
+    depends_on:
+      - insights-inventory-mq
+    ports:
+      - 8081:8080
+
   puptoo:
-    image: quay.io/cloudservices/insights-puptoo:latest
+    image: quay.io/cloudservices/insights-puptoo:stable
```

Comment:
If the new sample report is working with puptoo, then can we keep it latest?

Reply:
Fixed